### PR TITLE
[TG-297] [refactor] remove TestBed from scenarious module

### PR DESCRIPTION
**Pull request by valakar**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/153" target="_blank">ReviewNinja

File: src/app/scenarios/scenario-details/editor/create-scenario-editor/create-scenario-editor.component.test.ts [hunk 1]
```diff
@@ -11,8 +10,11 @@ describe('CreateScenarioEditor', () => {
             iam: 'a scenario'
         };
 
-        scenarioService = jasmine.createSpyComponent(ScenarioService);
-        scenarioService.create.and.returnValue({ subscribe(fn) { fn(); } });
+        scenarioService = {
+            create: jasmine.createSpy('create').and.returnValue({
```
Comment (velmyk) [inline src/app/scenarios/scenario-details/editor/create-scenario-editor/create-scenario-editor.component.test.ts]: !resolve I suggest to use real Observables in tests

File: src/app/scenarios/scenarios-list/scenario-list.component.test.ts [hunk 2]
```diff
@@ -119,7 +70,7 @@ describe('ScenarioListComponent', () => {
         it('should update scenario status on event', () => {
             const pausedState = false;
             sut.scenarioList = listData;
-            scenarioStatusService.stream.next({id: listData[1].id, active: pausedState});
+            ScenarioStatusService.stream.next({id: listData[1].id, active: pausedState});
 
             expect(sut.scenarioList[1].active).toBe(pausedState);
```
Comment (velmyk) [inline src/app/scenarios/scenarios-list/scenario-list.component.test.ts]: !resolve rePlease check one assertion per test

File: src/app/scenarios/scenario-details/editor/create-scenario-editor/create-scenario-editor.component.test.ts (hunk 1 above)
Comment (valakar) [inline src/app/scenarios/scenario-details/editor/create-scenario-editor/create-scenario-editor.component.test.ts]: !resolved

File: src/app/scenarios/scenarios-list/scenario-list.component.test.ts (hunk 2 above)
Comment (valakar) [inline src/app/scenarios/scenarios-list/scenario-list.component.test.ts]: !resolved

**Pull request merged**